src/i18n/index.jsx

```diff
@@ -9,6 +9,9 @@ import deMessages from './messages/de.json';
 import ruMessages from './messages/ru.json';
 import hiMessages from './messages/hi.json';
 import frCAMessages from './messages/fr_CA.json';
+import dedeCAMessages from './messages/de_DE.json';
```

Comment:
I do not believe the "CA" should be in these imports.  I presume this was inherited from the frCA line above, but the CA in that case stands for "Canadian > French".  Here, we do not have a sub-country language, so the CA is inappropriate.